python/metatensor-learn/tests/module_map.py

```diff
@@ -87,3 +96,84 @@ def test_module_map_single_block_tensor(single_block_tensor_torch, out_propertie
                 )
             else:
                 assert out_block.gradient(parameter).properties == out_properties[0]
+
+
+@pytest.mark.parametrize(
+    "out_properties", [None, [Labels(["a", "b"], np.array([[1, 1]]))]]
+)
+def test_module_map_to(single_block_tensor_torch, out_properties):  # noqa F811
+    """
+    Checks the `to` function of module map
+    """
+    modules = []
+    for key in single_block_tensor_torch.keys:
+        modules.append(
+            MockModule(
+                in_features=len(single_block_tensor_torch.block(key).properties),
+                out_features=5,
+            )
+        )
+    tensor_module = ModuleMap(
+        single_block_tensor_torch.keys, modules, out_properties=out_properties
+    )
+    tensor_module.to("cpu")
```

Comment:
this does nothing, right? Since everything is already on CPU

Reply:
Yes, you're right, I will try meta